tests/playbooks/kubevirt_vm_multus.yml

```diff
@@ -0,0 +1,65 @@
+- name: Create Fedora vm with two network interfaces
+  hosts: localhost
+  gather_facts: false
+  connection: local
+
+  # Before running this playbook you need to make sure you have created
+  # 'mynetconf' NetworkAttachmentDefinition. This is used by the second
```

Comment:
Should we provide a link to documentation how to do it?

Reply:
The only we have is following:

 https://kubevirt.io/user-guide/#/workloads/virtual-machines/interfaces-and-networks?id=multus

I will paste it.

Reply:
OK